### DISCUSSION TITLE
Just a Future Question

### DISCUSSION THREAD

**Discussion by Ada3181** [ExistentialAudio/BlackHole#507] [Q&A] [answered]
Hi! I've been using Blackhole on MacOS BigSur for around a month now, but of course as soon as I start actually recording stuff on OBS with it, Apple began pushing its Monterey update. I know Blackhole isn't usable, I guess is the word, for Monterey right now, so I've refrained from updating my laptop. At some point in the future however, do you think Blackhole would be usable for Monterey OS?

**Comment by devinroth** [comment, marked as answer]
It works fine for me. I have not been able to reproduce any of the bugs people have mentioned.

Devin

**Reply by Sleightly2015** [reply]
Following up on this.

Does BlackHole still work properly on MacOS Monterey after all of its updates? And if so, will you update the compatibility listed under Features to reflect this?

Thanks!

**Comment by devinroth** [comment]
Yes it does. And yes I still need to update the websites.

Devin
